app/scripts/services/events.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+angular.module('openshiftConsole').factory('EventsService', function() {
+
+  // TODO: decide if sessionStorage/localStorage should just be a service itself
+  var cacheKey = 'events-read';
```

Comment:
Agree. I'd like to break this out into a storage service that automatically does that, if no objections?  This first 12 lines could easily be reused elsewhere.